libs/pages/services/src/lib/ui/table-row-services/table-row-services.tsx

```diff
@@ -45,33 +46,39 @@ export function TableRowServices(props: TableRowServicesProps) {
     isLoading,
   } = props
 
+  const dataDatabase = data as DatabaseEntity
+  const dataApplication = data as GitApplicationEntity
+  const dataContainer = data as ContainerApplicationEntity
+  const dataJobs = data as JobApplicationEntity
+
   return (
     <TableRow data={data} filter={filter} columnsWidth={columnsWidth} link={link}>
       <>
         <div className="flex items-center px-4 gap-1">
-          {(data as DatabaseEntity).mode === DatabaseModeEnum.MANAGED ? (
+          {dataDatabase.mode === DatabaseModeEnum.MANAGED ? (
             <Skeleton show={isLoading} width={16} height={16} rounded={true}>
               <StatusChip status={data.status?.state} />
             </Skeleton>
           ) : (
             <Skeleton className="shrink-0" show={isLoading} width={16} height={16}>
-              {(data as DatabaseEntity).mode === DatabaseModeEnum.MANAGED ? (
-                <StatusChip status={data.status?.state} />
-              ) : (
-                <StatusChip
-                  status={data.running_status?.state || RunningStatus.STOPPED}
-                  appendTooltipMessage={
-                    data?.running_status?.state === RunningStatus.ERROR
-                      ? data.running_status.pods[0]?.state_message
-                      : ''
-                  }
-                />
-              )}
+              <StatusChip
+                status={data.running_status?.state || RunningStatus.STOPPED}
+                appendTooltipMessage={
+                  data?.running_status?.state === RunningStatus.ERROR ? data.running_status.pods[0]?.state_message : ''
+                }
+              />
             </Skeleton>
           )}
           <div className="ml-2 mr-2">
             <Skeleton className="shrink-0" show={isLoading} width={16} height={16}>
-              <Icon name={type === ServiceTypeEnum.DATABASE ? IconEnum.DATABASE : IconEnum.APPLICATION} width="20" />
+              <Icon
+                name={
+                  type === ServiceTypeEnum.APPLICATION || type === ServiceTypeEnum.CONTAINER
```

Comment:
With this multiplication of rather complex tests here and there, I think even more about the util that contains isContainer() isGitApplication() isDatabase() isJob()
It would be much comfortable to read and to maintain no?